main.py

```diff
@@ -1,29 +1,18 @@
-import json
 import pandas as pd
-from utils import save_values, dataframe_3D_to_2D
+from utils import save_values, dataframe_3D_to_2D, check_user_input
 from aggregation import Version3SafeAggregation
 
 
 def apply_secret_stat(group_by: list,
                       columns_apply_secret: list,
-                      column_to_check: str,
+                      column_to_check: str = None,
                       data_path: str = None,
                       sep: str = "|",
                       dataframe: pd.DataFrame = None,
                       export_to_csv: bool = False,
                       path_to_export: str = "./") -> dict:
-    if data_path is None:
-        if dataframe is None:
-            exit("specify data in order to process")
-        df_entreprises = dataframe
-    else:
-        if dataframe:
-            exit("To many data provided")
 
-        with open("config.json") as f:
-            config = json.load(f)
-        # Importing Dataset
-        df_entreprises = pd.read_csv(data_path, encoding='utf-8', sep=sep)
+    df_entreprises = check_user_input(data_path, dataframe, sep, column_to_check)
```

Comment:
rename : df_entreprise n'a pas de sens dans un contexte de package
```suggestion
    _df = check_user_input(data_path, dataframe, sep, column_to_check)
```